Cargo.toml

```diff
@@ -8,6 +8,13 @@ members = [
 	"errno",
 	"net",
 ]
+
+default-members = [
```

Comment:
Is this type of build even valuable at this point?  We aren't building dataplane which is the main thing we want to build, no dpdk, etc.

Reply:
yes it is useful for unit testing and aarch64 environments